includes/blocks/subtitle-block/post-editor.js

```diff
@@ -0,0 +1,81 @@
+/* globals newspack_block_theme_subtitle_block */
+'use strict';
+
+/**
+ * WordPress dependencies
+ */
+import { registerPlugin } from '@wordpress/plugins';
+import { withSelect, withDispatch } from '@wordpress/data';
+import { compose } from '@wordpress/compose';
+import { useEffect } from '@wordpress/element';
+
+const META_FIELD_NAME = newspack_block_theme_subtitle_block.post_meta_name;
+
+const SUBTITLE_ID = 'newspack-post-subtitle-element';
+const SUBTITLE_STYLE_ID = 'newspack-post-subtitle-element-style';
+const appendSubtitleToTitleDOMElement = ( subtitle, callback ) => {
+	const titleWrapperEl = document.querySelector( '.edit-post-visual-editor__post-title-wrapper' );
+
+	if ( titleWrapperEl && typeof subtitle === 'string' ) {
+		let subtitleEl = document.getElementById( SUBTITLE_ID );
+		const titleParent = titleWrapperEl.parentNode;
+
+		if ( ! document.getElementById( SUBTITLE_STYLE_ID ) ) {
+			const style = document.createElement( 'style' );
+			style.innerHTML = `
+                #${ SUBTITLE_ID } {
+                    font-style: italic;
```

Comment:
The italic font style is not reflected in the front-end, is this on purpose?

| Editor | Frontend |
| --- | --- |
| <img width="722" alt="image" src="https://github.com/user-attachments/assets/0b298bd9-5e07-4619-8174-2b360bd277dd"> | <img width="736" alt="image" src="https://github.com/user-attachments/assets/b8056a1b-7e08-4b02-b3b4-d1c70142df5c"> |

If that's a strategy to differentiate the subtitle from post content, maybe we can use something else, like a border-bottom. Wdyt?

Reply:
Fixed in 1cccf8095f6a74c37134ce5c9a05b52a612d2f86

> If that's a strategy to differentiate the subtitle from post content, maybe we can use something else, like a border-bottom. Wdyt?

For now it's just about matching what's in the Newspack Theme.